pyproject.toml

```
@@ -19,7 +19,7 @@ license = { file = "LICENSE" }
 authors = [{ name = "Trail of Bits", email = "opensource@trailofbits.com" }]
 dependencies = [
     "maturin>=1.7,<2.0",
-    "cryptography>=43,<44",
+    "cryptography>=43,<45",
```

Our downstream uses are all going to be `>=44`, so let's just constrain to that.

```suggestion
    "cryptography>=44,<45",
```